modules/ROOT/pages/common/nav.adoc

```diff
@@ -57,11 +57,11 @@
 **** link:{{navprefix}}/events-app-integration[Events and app interactions]
 **** link:{{navprefix}}/custom-action-intro[Custom actions]
 ***** link:{{navprefix}}/customize-actions[Create and manage custom actions]
+***** link:{{navprefix}}/custom-action-callback[Callback actions]
 ***** link:{{navprefix}}/edit-custom-action[Set the position of a custom action]
 ***** link:{{navprefix}}/add-action-viz[Add a local action to a visualization]
 ***** link:{{navprefix}}/add-action-worksheet[Add a local action to a worksheet]
 ***** link:{{navprefix}}/custom-action-url[URL actions]
```

Comment:
I think we should place URL actions above "Set position of custom action" and "Callback response payload" below "Callback actions".

Reply:
No one ever uses URL actions because of a some flaws in the design, that's why I have them placed so low